packages/o/objfw/xmake.lua

```diff
@@ -12,13 +12,47 @@ package("objfw")
         add_frameworks("CoreFoundation")
     end
 
-    on_install("linux", "macosx", function (package)
-        local configs = {"--without-tls"}
+    add_configs("tls", {
+        description = "Enable TLS support.",
+        default = true,
+        type = "boolean"
+    })
+
+    add_configs("arc", {
+        description = "Enable Automatic Reference Counting (ARC) support.",
+        default = true,
+        type = "boolean"
+    })
+
+    on_install("linux", "macosx", "cygwin", function (package)
+        local configs = {}
+        table.insert(configs, (package:config("tls") and "" or "--without-tls"))
         table.insert(configs, "--enable-shared=" .. (package:config("shared") and "yes" or "no"))
-        if package:debug() then
+        if package:is_debug() then
             table.insert(configs, "--enable-debug")
         end
         import("package.tools.autoconf").install(package, configs)
+
+        local tool = import("lib.detect.find_tool")
```

Comment:
remove `local tool = ` and use `find_tool`

Reply:
kk